server/controllers/lists/remove_selections.js

```diff
@@ -0,0 +1,36 @@
+const _ = require('builders/utils')
+const lists_ = require('controllers/lists/lib/lists')
+const { filterFoundSelectionsUris } = require('controllers/lists/lib/helpers')
+const error_ = require('lib/error/error')
+const selections_ = require('controllers/lists/lib/selections')
+const { addWarning } = require('lib/responses')
+
+const sanitization = {
+  id: {},
+  uris: {}
```

Comment:
i imagined the client only sending uris and server knows what to do (but since client is not coded yet, it might be too far fetched)
I dont think its less future proof, since, in the case you mention, extending API to accept a selectionId params should be enough for clients.
maybe we could merge like this now, and add an update after client implementation?